.github/labeler.yml

```diff
@@ -0,0 +1,30 @@
+documentation:
+  - changed-files:
+    - any-glob-to-any-file:
+      - 'README*'
+      - 'CONTRIBUTING*'
+      - 'code-of-conduct*'
+
+tests:
+  - changed-files:
+    - any-glob-to-any-file:
+      - 'e2e-tests/**'
+      - 'Jenkinsfile*'
+
+ci-cd:
+  - changed-files:
+     - any-glob-to-any-file: '.github/workflows/**'
+
+dockerfile:
+  - changed-files:
+    - any-glob-to-any-file: 'build/Dockerfile'
+
```

Comment:
maybe we can call this `build` and match all `build/*?

Reply:
I had the same idea :)

Reply:
[7a5d7ee](https://github.com/percona/percona-server-mongodb-operator/commit/7a5d7ee2b2631682ee73f4c6824eb15e73397204)